### PR TITLE
[MODULAR] Lewd Clothing Tweaks

### PR DESCRIPTION
**Pull request by Mryourface13**
## About The Pull Request
1. Allows you to lace and unlace corset in order to control slowdown manually. 0/1
2. Replaces the Uniform Corset in the Lustwish with Suit Corset.
## How This Contributes To The Skyrat Roleplay Experience
Now you can use the correct corset over your uniform, and also lace it if you want to move slightly slower for cinematic reasons.

## Changelog
:cl:Mr_yourface13
add: Added suit corset in Lustwish.
tweak: Renamed suit Corset.
add: Provided corporate training for walking faster in Corsets. We also added laces.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#define TIGHT_SLOWDOWN 2
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm:1]: keep the define but set it to 1

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm [hunk 2]
```diff
@@ -14,7 +13,7 @@
 	lefthand_file = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_inhands/lewd_inhand_left.dmi'
 	righthand_file = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_inhands/lewd_inhand_right.dmi'
 	body_parts_covered = CHEST
-	slowdown = 1 // You can't run with that thing literally squeezing your chest
+	slowdown = 0 // No longer slows down while not laced
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm]: this line can be removed completely

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm [hunk 3]
```diff
@@ -38,4 +39,6 @@
 	if(laced_tight && src == user.wear_suit)
 		to_chat(user, span_purple("Phew. Now you can breathe normally."))
 
-#undef TIGHT_SLOWDOWN
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm:41]: keep the undef

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm [hunk 4]
```diff
@@ -38,4 +39,6 @@
...
+/obj/item/clothing/suit/corset/examine(mob/user)
+	. = ..()
+	. += span_notice("Alt-Click to [laced_tight ? "unlace" : "lace"] the corset.")
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/corset.dm]:
```suggestion
	. += span_notice("<b>Alt-Click</b> to [laced_tight ? "unlace" : "lace"] [src].")
```

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/lewd_shoes.dm [hunk 5]
```diff
@@ -1,7 +1,7 @@
 //heels item
 /obj/item/clothing/shoes/latexheels
 	name = "latex heels"
-	desc = "Lace up before use. It's pretty difficult to walk in these."
+	desc = "Lace up before use. It's pretty difficult to walk in these while they are laced."
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/lewd_shoes.dm]: already told you on discord but saying here for the record, no readding mechanics to the just mechanic-removed heels

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by YakumoChen** [comment]
> Replaces the Uniform Corset in the Lustwish with Suit Corset.

why not both?

**Comment by Mryourface13** [comment]
> 

They're named the exact same thing so it could cause confusion. If you have a suggestion for renaming one or the other, I'm all ears.

**Comment by YakumoChen** [comment]
name one overwear corset or something more obvious

**Pull request closed without merging**